packages/aries-hooks/README.md

```diff
@@ -0,0 +1,68 @@
+# Aries Hooks
+
+This package exposes useful hooks that allow you to easily interact with AFJ.
+
+Everything exported from Aries Hooks:
+
+```
+import
+	AgentProvider, {
+	useAgent,
+	useConnections,
+	useConnectionById,
+	useConnectionByState,
+	useCredentials,
+	useCredentialById,
+	useCredentialByState,
+	useProofs,
+	useProofById,
+	useProofByState
+} from 'aries-hooks'
+```
+
+First step is to wrap your entire app in our `<AgentProvider/>`. The provider takes two props, the first is your `agentConfig` object and the second is your `genesisUrl`. The base of your app should look something like this:
+
+```
+import AgentProvider from "aries-hooks"
+
+const App = () => {
+	return (
+		<AgentProvider agentConfig={Your full agent config object} genesisUrl={Your genesis url} >
```

Comment:
Do we want to do the genesis via an url? I am not a 100% sure it is a must feature. It could always be handled like this by the framework consumer.